src/openvr_api_public.cpp

```diff
@@ -1,5 +1,6 @@
 //========= Copyright Valve Corporation ============//
 #define VR_API_EXPORT 1
+#define VR_API_PUBLIC 1
```

Comment:
Can you elaborate on what this does? It appears to exclude a function call to `CleanupInternalInterfaces` that isn't implemented anywhere.

Reply:
I could not build the DLL otherwise, since `CleanupInternalInterfaces` is not defined and could not be linked.
```
openvr_api_public.obj : error LNK2019: unresolved external symbol "void __cdecl vr::CleanupInternalInterfaces(void)" (?Cleanu
pInternalInterfaces@vr@@YAXXZ) referenced in function VR_ShutdownInternal [C:\Steam\steamapps\common\!openvr_fsr_v2.0\openvr_
fsr-patch-1\src\_build\openvr_api.vcxproj]
C:\Steam\steamapps\common\!openvr_fsr_v2.0\openvr_fsr-patch-1\src\_build\Debug\openvr_api.dll : fatal error LNK1120: 1 unreso
lved externals [C:\Steam\steamapps\common\!openvr_fsr_v2.0\openvr_fsr-patch-1\src\_build\openvr_api.vcxproj]
```
See my comment in [issue #59](https://github.com/fholger/openvr_fsr/issues/59#issuecomment-991809077).

Reply:
The weird thing is, this option should be set by CMake. There's a `add_definitions( -DVR_API_PUBLIC )` in the CMakeLists.txt.